clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AvroGenericStoreClient.java

```diff
@@ -53,6 +53,7 @@ default CompletableFuture<Map<K, V>> batchGet(Set<K> keys) throws VeniceClientEx
             + "or VeniceGenericStoreClientFactory#createInstance() to generate a Venice avro generic client");
   }
 
+  // TODO: Should streamingBatchGet() be exposed once batchGet() defaults to start using streamingBatchGet()
```

Comment:
Yes they have different guarantees. batchGet will block and return only when all key values have been retrieved

Reply:
But I think the plan is to default the batchGet() call to use streamingBatchGet() internally, similar to thin client. Marking this as resolved and will be revisited when making this actual API change.